sdks/builds/desktop.mk

```diff
@@ -0,0 +1,71 @@
+
+##
+# Parameters:
+#  $(1): arch
+#  $(2): host_triple
+#
+# Flags:
+#  desktop_$(1)_CFLAGS
+#  desktop_$(1)_CXXFLAGS
+#  desktop_$(1)_LDFLAGS
+define DesktopTemplate
+
+#_desktop_$(1)_AC_VARS= \
```

Comment:
You should remove commented out code, as it seems to be a leftover from the copy-paste from ios.mk